quantumarticle.cls

```diff
@@ -216,7 +245,23 @@
 \@medpenalty  151
 \@highpenalty 301
 
-\AtBeginDocument{
+\AfterEndPreamble{
+	% avoid use of today	
+	\ifcsdef{today}
+	{
+		\ifbool{allowtoday}
+		{
+			\let\oldtoday\today
+			\renewcommand*{\today}{\oldtoday\ClassWarning{quantumarticle}{If you intend to upload your paper to the arxiv, please replace any use of today with the publishing date}{}}
+		} % else
+		{	
+			\renewcommand*{\today}{\ClassError{quantumarticle}{Use of the today macro should be omitted in papers that are to be uploaded to the arxiv. If you don't intend to upload your paper to the arxiv use the class option allowtoday}{}}
```

Comment:
I have deliberately kept the second one as a simple warning, because it will only be activated if the user has chosen to use the option 'allowtoday'.

> Thanks! This is looking very good!
> 
> One comment on the workflow: In the future I would like you to create a new branch, branching off from `develop` for every fix/new feature and then submit a PR against develop, instead of pushing directly to develop. I didn't explain that, so don't worry, this was entirely my fault.
> 
> The rational behind that is that I would like `master` to always reflect the current "stable" release and `develop` the current "release candidate". Creating a branch for each feature seems cumbersome at first, but it makes keeping track of what has been fixed how and when easier.
> 
> I suggest we make an exception in this case and you implement the requested changes on this PR directly in develop, so that we can use the code review and comment feature of this PR. In the end I will not merge the PR, but leave the changes in `develop` until we are ready to release. Does that make sense?
> 
> Thanks again!

Totally understand that! Yesterday I talked to some people and they said this is how you do it. Should i then just make a branch "jmeyer" or do I call it differently? Is there some kind of best practice for that?

Reply:
Please keep it a Warning. I forgot to delete that line from my suggested change, sorry....

Reply:
It is best to name the branches according to the issue the fix or the feature they add, i.e., this one could have been named `discourage_today`, or something like that.

Reply:
It is good to be rather "fine grained" with this, so that changesets that address different things end up in different branches and ultimately differente pull requests (PRs).

Reply:
And I don't get the 'suggested change' window, did you propose two different changes for the same line? Why can't I expand the window to see the surroundings of this line?

Reply:
It seems like github couldn't handle my two suggestions at once well. I deleted one. Is the remaining one now clear?